extensions/wikia/SpecialPromote/UploadVisualizationImageFromFile.class.php

```diff
@@ -57,7 +57,10 @@ public function checkWarnings(){
 	static public function UploadVerification($destName, $tempPath, &$error) {
 		$promoImage = PromoImage::fromPathname($destName);
 
-		if($promoImage->isValid() and ($promoImage->getCityId() == F::app()->wg->cityId)){
+		if ( $promoImage->isValid() &&
+			$promoImage->getCityId() == F::app()->wg->cityId ||
+			!PhalanxTitleBlock::checkTitle( Title::newFromText( $destName ), true )
```

Comment:
OK, I'll add them, may not be obvious to everyone. But I think that adding parentheses around `==` like it was previously (BTW why not `===`?) is overcorrect.